rskj-core/src/main/java/co/rsk/peg/feeperkb/FeePerKbStorageProvider.java

```diff
@@ -0,0 +1,61 @@
+package co.rsk.peg.feeperkb;
+
+import co.rsk.bitcoinj.core.Coin;
+import co.rsk.peg.*;
+import co.rsk.peg.abi.ABICallElection;
+import org.ethereum.core.Repository;
+
+import static co.rsk.peg.BridgeStorageIndexKey.FEE_PER_KB_ELECTION_KEY;
```

Comment:
We should probably have an enum `FeePerKbStorageKey`, what do you think?

Reply:
Yes! totally agree. Thanks!